engine/apps/alerts/incident_appearance/templaters/phone_call_templater.py

```diff
@@ -14,14 +14,11 @@ def _postformat(self, templated_alert):
         return templated_alert
 
     def _postformat_pipeline(self, text):
-        return self._escape(clean_markup(self._slack_format_for_phone_call(text))) if text is not None else text
+        return clean_markup(self._slack_format_for_phone_call(text)) if text is not None else text
 
     def _slack_format_for_phone_call(self, data):
         sf = self.slack_formatter
         sf.user_mention_format = "{}"
         sf.channel_mention_format = "#{}"
         sf.hyperlink_mention_format = "{title}"
         return sf.format(data)
-
-    def _escape(self, data):
-        return escape_for_twilio_phone_call(data)
```

Comment:
Is that expected that we don't escape `< > &` anymore

https://github.com/grafana/oncall/pull/1713/files#diff-007b962c05ebca7bac34170b3d6fa519225c2bc6d652371dfc27ae018f942b73L193-L195

Reply:
Yes, this escape was TwilioRelated, so it's moved to TwilioPhoneProvider https://github.com/grafana/oncall/pull/1713/files#diff-764f1941b04b97b1517990791c24bc6446e623f0326b95c44060dd4ec42afd75R26